DataLayer/DriverControlsData/DriverControlsData.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "I_DriverControlsData.h"
+
+class DriverControlsData : public I_DriverControlsData
+{
+public:
+    DriverControlsData();
+    virtual ~DriverControlsData();
+
+    /*Data "Gets"*/
+    unsigned char getDriverControlsBoardAlive() const;
+    unsigned char getLightsInputs() const;
+    unsigned char getMusicInputs() const;
+    unsigned short getAcceleration() const;
+    unsigned short getRegenBraking() const;
+    unsigned char getDriverInputs() const;
+
+    /*DriverControlsData "Sets"*/
+    void setDriverControlsBoardAlive(const unsigned char& driverControlsBoardAlive);
+    void setLightsInputs(const unsigned char& lightsInputs);
+    void setMusicInputs(const unsigned char& musicInputs);
+    void setAcceleration(const unsigned short& acceleration);
+    void setRegenBraking(const unsigned short& regenBraking);
+    void setDriverInputs(const unsigned char& driverInputs);
+
+private:
+    unsigned char driverControlsBoardAlive_;
+    unsigned char lightsInputs_;
+    unsigned char musicInputs_;
```

Comment:
These shoudln't be bitflags now that they're in the data layer.  They should just be boolean variables for each field.